src/SyncTrayzor/Pages/ConsoleViewModel.cs

```diff
@@ -17,14 +19,28 @@ public class ConsoleViewModel : Screen
 
         public ObservableQueue<string> LogMessages { get; private set; }
 
-        public ConsoleViewModel(ISyncThingManager syncThingManager)
+        public ConsoleViewModel(
+            ISyncThingManager syncThingManager,
+            IConfigurationProvider configurationProvider
+            )
         {
+            var configuration = configurationProvider.Load();
```

Comment:
Actually - this won't pick up changes in config without a restart. You'll have to subscribe to IConfigurationProvider.ConfigurationChanged. Most 'reload when config changed' is handled by the `ConfigurationApplicator`, but that won't work with the ViewModels, as they're not singleton (and nor should they be).